firefox-ios/Client/Frontend/Library/Bookmarks/Legacy/LegacyBookmarkDetailPanel.swift

```diff
@@ -333,6 +334,18 @@ class LegacyBookmarkDetailPanel: SiteTableViewController, BookmarksRefactorFeatu
         return url?.schemeIsValid == true && url?.host != nil
     }
 
+    private func dismissKeyBoardWhenTapOnOutsideTapGesture() {
```

Comment:
Thanks so much for this fix @gokulvenkat243 🙏🏼
Do you think you could rename this method to `addDismissKeyboardGesture()`? The current name sounds more like the action executed when the gesture is activated.